client/packages/system/src/Patient/CreatePatientModal/CreatePatientModal.tsx

```diff
@@ -128,7 +128,7 @@ export const CreatePatientModal: FC<CreatePatientModal> = ({ onClose }) => {
       okButton={
         currentTab === Tabs.SearchResults ? (
           <DialogButton
-            variant="create"
+            variant="next"
```

Comment:
Quick roll of the eyes 🙄 
#728 was when it was changed from `ok` to `create` because the ok text didn't indicate clearly enough that you will be creating a patient. I suppose that `ok & next` is more of an indication that something is going to happen, and that you won't be creating the patient yet.

Reply:
ups :) I found it confusing that create didn't save the patient directly.

We could save the patient directly though. The reason why its not done is that you would have two document versions when creating a patient, one with the base information and one with the full information from the full form. Probably not an issue to have two version immediately though...